helix-term/src/health.rs

```diff
@@ -279,6 +285,13 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
         }),
     )?;
 
+    probe_protocol(
+        "formatter",
+        lang.formatter
+            .as_ref()
+            .map(|formatter| formatter.command.to_string()),
+    )?;
```

Comment:
Let's move this after the debug adapter so the order matches the overall `--health` display

Reply:
Thanks for reviewing @the-mikedavis , just pushed to move this as you have suggested.

```
Configured language server: pylsp
Binary for language server: /opt/homebrew/bin/pylsp
Configured debug adapter: None
Configured formatter: black
Binary for formatter: /opt/homebrew/bin/black
Highlight queries: ✓
Textobject queries: ✓
```